docs/src/content/docs/ru/resources/community-content.mdx

```diff
@@ -50,6 +50,11 @@ import { CardGrid, LinkCard } from '@astrojs/starlight/components';
 		title="Starlight Changelog"
 		description="Просмотр всех релизов пакета @astrojs/starlight на одной удобной странице."
 	/>
+	<LinkCard
+        href="https://hideoo.dev/notes/contributing-to-starlight"
+        title="Contributing to Starlight"
```

Comment:
Just to double check, was the title left in English on purpose or potentially something missed during the translation?